packages/shared_preferences/shared_preferences_web/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.0.1
```

Comment:
pub.dev is going to ding you for having such a low release number, I'd recommend bumping to at least `0.1.0` before publishing.

Reply:
Done.